elementary/monitor/data_monitoring/alerts/integrations/slack/message_builder.py

```diff
@@ -78,6 +78,12 @@ def add_details_to_slack_alert(
             ]
             self._add_blocks_as_attachments(configuration_blocks)
 
+    def add_color_to_slack_alert(self, color: str):
+        for block in self.slack_message.get("blocks", []):
```

Comment:
I know that is not part of this change, but this is small and could be really nice for the future, so if you feel like doing so it would be great!

Maybe we can add a @property to the `SlackMessageBuilder` for `blocks` and `attachments` that returns `slack_message.get("blocks", [])/slack_message.get("attachments", [])`